app/src/main/java/io/intrepid/contest/screens/sendinvitations/selectcontacts/SelectContactsViewHolder.java

```diff
@@ -51,11 +54,19 @@ public void bindData(Contact contact) {
             emailTextView.setVisibility(GONE);
         }
 
-        byte[] photo = contact.getPhoto();
-        if (photo == null) {
-            photoImageView.setImageResource(R.drawable.default_user_photo);
+        if (contact.isSelected()) {
+            photoImageView.setImageResource(R.drawable.selected_contact_icon);
         } else {
-            photoImageView.setImageBitmap(BitmapFactory.decodeByteArray(photo, 0, photo.length));
+            byte[] photo = contact.getPhoto();
+            if (photo == null) {
+                photoImageView.setImageResource(R.drawable.default_user_photo);
+            } else {
+                photoImageView.setImageBitmap(BitmapFactory.decodeByteArray(photo, 0, photo.length));
+            }
         }
     }
+
+    public interface ContactClickListener {
+        void onContactClick(View itemView);
```

Comment:
I actually don't get the contact from the viewholder or the adapter; I get it from the presenter's contact list the contact in the same position as the `itemView`.
I could send in just the position, but I feel like it should be my presenter deciding what it wants from that view?
Good point though